pkg/ccl/backupccl/restore_job.go

```diff
@@ -1172,22 +1196,6 @@ func createImportingDescriptors(
 		return nil, nil, err
 	}
 
-	// Collect all schemas that are going to be restored.
```

Comment:
Is there any reason this code was moved? If not, can we move it back to reduce churn.

Reply:
Oops sorry this has to be above `RewriteTableDescs` since
rewriteTableDescs below it relies on the parentSchemaID being correct
```
		table.ID = tableRewrite.ID
		table.UnexposedParentSchemaID = tableRewrite.ParentSchemaID
		table.ParentID = tableRewrite.ParentID
```

Reply:
thanks!